spec/migrations/20210330012001_remove_regex_from_settings_spec.rb

```diff
@@ -0,0 +1,24 @@
+require_migration
+
+# Vmdb::Settings.save!(MiqServer.my_server, {ems: {ems_nuage: { event_handling: {event_groups: {addition: {critical: /abc/}}}}}})
+# Settings[:ems][:ems_nuage][:event_handling][:event_groups][:addition][:critical]
+# Settings.reload! ; nil
+
+describe RemoveRegexFromSettings do
+  let(:settings_change_stub) { migration_stub(:SettingsChange) }
+
+  migration_context :up do
+    it "converts regular expression object to a string" do
+      record = settings_change_stub.create!(
+        :resource_type => "MiqServer",
+        :key           => "/ems/ems_nuage/event_handling/event_groups/addition/critical",
+        :value         => ["/hello/","beautiful",/world/]
```

Comment:
@kbrock is this actually a Regexp object?

Reply:
the first is a string w/ a regex (no change), the second is a string (no change) and the third is a regex (change)

Reply:
🤦 sorry didn't see the last one didn't have `"` around it like the others

Reply:
@agrare do you think I should document this - are the 3 use cases too sneaky?